docs/hw/display/nv3/pcrtc.rst

```diff
@@ -26,8 +26,9 @@ MMIO registers
 
 .. space:: 8 prmcio 0x1000 VGA CRTC & attribute controller registers
 
-   .. todo:: write me
-
+   0x3d0 RMA_ACCESS nv3-io-rma-access
+   0x3d4 CRTC_INDEX nv3-io-crtc-index
+   0x3d5 CRTC_DATA nv3-io-crtc-data
 
```

Comment:
Please leave the TODO items here (or change to "complete me"), since it's not the complete register set (at least 0x3c0 is missing)